Docs/development/r-analyses-guide.md

```diff
@@ -929,19 +929,19 @@ We can now give the text to `jaspResults` to display.
 	<summary>Code</summary>
 
   ```r
-  jaspResults[["textDescriptives"]] <- textDescriptives
+  jaspResults[["binomTextDescriptives"]] <- binomTextDescriptives
   ```
 
 </details>
 
 #### Step 5.3.4 - Formatting the Text
-Text in JASP can be formatted using HTML tags to highlight certain parts. The following code shows an example. Here, the text enclosed between \<b\> and \<\/b\> will be **bold**, and the text between \<i\> and \<\/i\> will be <i>italicized</i>.
+Text in JASP can be formatted using [HTML tags](https://www.w3schools.com/html/html_formatting.asp) to highlight certain parts. The following code shows an example. Here, the text enclosed between \<b\> and \<\/b\> will be **bold**, and the text between \<i\> and \<\/i\> will be <i>italicized</i>.
 
 <p><details>
 	<summary>Code</summary>
 
   ```r
-	textDescriptives <- createJaspHtml(text = gettextf("If the number of <b>defective</b> items out of %d <i>sampled</i> is <= %d, accept the lot. Reject otherwise.", 50, 3))
+	binomTextDescriptives <- createJaspHtml(text = gettextf("The variable <b>%s</b> has more than <i>%d</i> unique values and is omitted.", variable, 2))
```

Comment:
```suggestion
  binomTextDescriptives <- createJaspHtml(text = gettextf("The variable <b>%1$s</b> has more than <i>%2$d</i> unique values and is omitted.", variable, 2))
```
nitpick: in R code we use spaces so the docs should do so too. Also added the `x$` to `gettextf`.